index.html

```diff
@@ -183,10 +183,43 @@ <h2>Skills</h2>
             </div>
           </div>
         </div>
-        <span class="chat">
-          I'm always <br> interested in <br> hearing about <br> new projects, so <br> if you'd like to <br> chat please get <br> in touch.
-        </span>
       </section>
     </main>
+    <footer class="form-section">
+      <p class="chat">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </p>
+      <form class="user-form" id="contactform" action="https://formspree.io/f/mbjwvgqk" method="post">
+        <input id="name" type="text" name="name" maxlength="30" placeholder="Full name" required>
+        <input id="email" type="email" name="_replyto" placeholder="Email address" required>
+        <textarea id="msg" name="message" maxlength="500" required>
```

Comment:
- [ ] The `textarea` should have a placeholder instead, kindly implement this as for now the form will submit even if the user didn't add a message because of `Enter text here` text